### ISSUE TITLE
Is this project maintained somewhere?

### ISSUE THREAD

**Issue by robguinness** [abbr/deasync#133]
This seems to be a great solution to a difficult problem, and I thank @abbr for the work. But is the project being maintained anymore? Issues are piling up, and there are seven pull requests that have not really been handled.

There are also a lot of forks, but unfortunately, github doesn't provide an easy way to see which of the forks are maintained. Anyone have suggestions how to keep this project maintained?

**Comment by papb** [comment]
Hello, I would suggest migrating to https://github.com/sindresorhus/make-synchronous and https://github.com/sindresorhus/sleep-synchronously instead

**Comment by NotWearingPants** [comment]
@papb While these suggestions are cool projects, they are very different to deasync, and I doubt most users of deasync could use these instead.

`deasync` - spins the event loop until the async action finishes while hanging the current task
`make-synchronous` - spawns a new node.js process using `spawnSync` to run your function (this means you can't really pass any non-trivial data such as database connections, or use any non-builtin module).

**Comment by orellabac** [comment]
Yes it is true 'deasync' is pretty unique. In my case I have to return to nodejs lts/dubnium 10.something.
I tried to determine possible causes of error but I am still lost. I see deasync used the deprecated _tickCallback method before calling uv_run.
I have not found anything weird now. If somebody is also looking into it and has any clues please add your findings in the comments.
